docs/src/content/docs/reference/frontmatter.md

```diff
@@ -224,3 +224,18 @@ sidebar:
   order: 1
 ---
 ```
+
+#### `hidden`
+
+**type:** `boolean`
+**default:** `false`
+
+Hides this page from being included in an autogenerated sidebar group.
+
+```md
+---
+title: Page to hide from autogenerated sidebar
+sidebar:
+  hidden: 1
```

Comment:
Shouldn't this be `true` ?

Reply:
good catch, copy paste got me on that one